pxr/imaging/plugin/hdRpr/materialFactory.cpp

```diff
@@ -121,7 +121,7 @@ RprApiMaterial* RprMaterialFactory::CreateMaterial(EMaterialType type, const Mat
 
         rpr_material_node materialNode = nullptr;
         rprMaterialSystemCreateNode(matSys, RPR_MATERIAL_NODE_IMAGE_TEXTURE, &materialNode);
-        rprMaterialNodeSetInputImageDataByKey(materialNode, RPR_MATERIAL_INPUT_DATA, img);
+        rprMaterialNodeSetInputImageDataByKey(materialNode, RPR_MATERIAL_INPUT_DATA, rprImage);
```

Comment:
This one is coming from https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/pull/88/commits/e7f9db4c25d2fcdb10f29d5d8c6506efd50d0011